src/ORM/Query.php

```diff
@@ -1146,11 +1147,14 @@ protected function _transformQuery()
             return;
         }
 
+        /** @var Table $repository */
```

Comment:
Should not be necessary due to the _repository property, right? But otherwise also FQCN.